message driverinfo {
    required uint32 version = 1;
    optional string dcn = 2;
}

message driverinforesponse {
    required uint32 version = 1;
    optional bool success = 2;
    
    optional string tune_type = 3;
    optional string stream_mimetype = 4;
}
